code/modules/uplink/uplink_items.dm

```diff
@@ -75,6 +75,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	var/player_minimum //The minimum crew size needed for this item to be added to uplinks.
 	var/purchase_log_vis = TRUE // Visible in the purchase log?
 	var/restricted = FALSE // Adds restrictions for VR/Events
+	var/list/restricted_species //Limits items to a specific species. Hopefully.
```

Comment:
This defaults to null instead of `list()` so this is guaranteed to cause the uplink to runtime.

Reply:
Well, you should've made the code in `uplink.dm` be able to accept null so it's still your fault.

Reply:
this is causing uplinks to fuck over on hippie; who would have guessed an PR made entirely out of webedits fucks over the uplink?

Reply:
#41597 
Tested this to confirm it works